src/snowcli/cli/streamlit.py

```diff
@@ -38,14 +38,6 @@ def get_standard_stage_name(name: str) -> str:
 def streamlit_list(
     environment: str = EnvironmentOption,
     only_cols: List[str] = typer.Option(list, help="Only show these columns"),
-    show_header: bool = typer.Option(
-        True,
-        help="Show column headers",
-    ),
-    show_border: bool = typer.Option(
-        True,
-        help="Show column borders",
-    ),
```

Comment:
Those were used by only once command. In future we can extend the output format to "prettier tables".